src/translate/pddl_parser/parsing_functions.py

```diff
@@ -317,7 +319,7 @@ def parse_domain_pddl(domain_pddl):
     iterator = iter(domain_pddl)
 
     define_tag = next(iterator)
-    assert define_tag == "define"
+    assert define_tag == "define", "define_tag = \"%s\" instead of \"define\"" % define_tag
```

Comment:
This is better than the current state but still not very obvious because it refers to the internals of the code. Maybe the text should rather be something along the lines of 'The domain specification must begin with "(define" but begins with "(%s".'

Reply:
To continue in this PR: https://github.com/aibasel/downward/pull/59